### PR TITLE
SL-19986 Ability to drag Favorites to the Overflow menu

### PR DESCRIPTION
**Pull request by LLGuru**

### CODE REVIEW

File: scripts/messages/message_template.msg.sha1 [hunk 1]
```diff
@@ -1 +1 @@
-d7915d67467e59287857630bd89bf9529d065198
+d7915d67467e59287857630bd89bf9529d065199
```
Comment (cosmic-linden) [inline scripts/messages/message_template.msg.sha1:1]: Why did the message template change?

File: indra/newview/llpanellandmarks.cpp [hunk 2]
```diff
@@ -1047,6 +1047,85 @@ bool LLLandmarksPanel::handleDragAndDropToTrash(BOOL drop, EDragAndDropType carg
...
+            item->setComplete(TRUE);
+            item->updateServer(FALSE);
+
+            gInventory.updateItem(item);
+            gInventory.notifyObservers();
+        }
+
+        LLView::getWindow()->setCursor(UI_CURSOR_ARROW);
+    }
+};
```
Comment (akleshchev) [inline indra/newview/llpanellandmarks.cpp:1070]: This looks incomplete, it's a landmark being copied into favorites' overflow, why isn't it calling setSortIndex to make sure position gets updated and saved?

Comment (akleshchev) [reply indra/newview/llpanellandmarks.cpp:1070]: I also don't see any code for a position indicator when hovering over the overflow menu (like it was done for the main favorites panel)

**Comment by akleshchev** [comment]
Since jira is now gone, please link development issues in the menu to the right.

**Review by akleshchev — CHANGES_REQUESTED** [review]
The whole point of the request was an ability to reorder favorites, what's the point if you aren't preserving new order acros sessions?

File: scripts/messages/message_template.msg.sha1 (hunk 1 above)
Comment (akleshchev) [reply scripts/messages/message_template.msg.sha1:1]: I think this one gets pulled automatically from server in some cases by TemplateCheck.cmake.

Comment (LLGuru) [reply scripts/messages/message_template.msg.sha1]: It was changed automatically during the compilation. Removed from PR, thanks

**Pull request closed without merging**